### ISSUE TITLE
[Bug]: Not showing fields in custom PDF after uploading PDF

### ISSUE THREAD

**Issue by bpbarwick** [arcanistzed/pdf-sheet#104]
### Expected Behavior

When uploading the 5e character sheet PDF, a window appears that shows all the fillable fields on the PDF. This is what I'm expecting to happen when working with my Quest RPG character sheet PDF.

### Current Behavior

When uploading the Quest RPG PDF, I see a blank window with none of the fillable fields present in the PDF. There is the print button at the bottom, but nothing else.

### Steps to Reproduce

1. Upload my Quest RPG character sheet PDF.
2. See the blank window.

### Context

The PDF I am working with is the official digital character sheet that I edited using Affinity Publisher. I then used [docfly.com](https://www.docfly.com/pdf-form-creator) to create the fillable fields.

I am using Brave browser.

[Quest Character Sheet.pdf](https://github.com/arcanistzed/pdf-sheet/files/12888074/Quest.Character.Sheet.pdf)

### Version

0.24.1

### Foundry VTT Version

11.313

### Operating System

Ubuntu

### Browser / App

Other

### Game System

Quest 0.2.2

### Modules Disabled

- [X] I can confirm that disabling all of my other modules doesn't fix this issue.

**Comment by gioppoluca** [comment]
If I understand well you are trying to use 5e mapping with a game system that is not 5e and with a PDF template that is not the one for 5e.
If this is correct, the behavior is not a bug since there is no mapping for the system you are trying to use and for sure not for the PDF you are trying to use so in this case is normal that it will not show anything.
You will need to create mapping for that system/pdf

**Comment by bpbarwick** [comment]
No. Sorry for the confusion. In my 5e world, the mod works great. I have another world for the Quest RPG system for which I have written a mapping system. If I use the official character sheet for Quest RPG, I can get it to work, but it does not have fillable fields for character abilities, so I created a a custom PDF form so my players could see this (its basically the only reason I wanted the printed sheet in the first place). When I upload the custom PDF, I get the attached screen.
![blank](https://github.com/arcanistzed/pdf-sheet/assets/67689531/14f5eb43-f552-46a1-af70-1e303a821e20)

**Comment by gioppoluca** [comment]
can you attach both mapping and pdf template you are using?

**Comment by bpbarwick** [comment]
Yes, and sorry I hadn't put it in the original post. This is my first time doing this... I appreciate your help. 

Here's the PDF (the weird fillable field naming convention is a carry over from the original sheet):
[Quest Character Sheet.pdf](https://github.com/arcanistzed/pdf-sheet/files/12905812/Quest.Character.Sheet.pdf)

Here's the mapping (I intend to add in ability descriptions to the paragraph fields, but have not gotten that far yet):

```
[    
    /* ---------- Biography ----------*/
        {"pdf": "HP", "foundry": @system.maxhp },
        {"pdf": "Name", "foundry": @name },
        {"pdf": "AP", "foundry": @system.ap },
        {"pdf": "Gender1", "foundry": @system.pronouns },
        {"pdf": "Age", "foundry": @system.age },
        {"pdf": "Height", "foundry": @system.height },
        {"pdf": "Clothing1", "foundry": @system.role },
        {"pdf": "Notes", "foundry": @system.notes },
        {"pdf": "undefined_13", "foundry": @system.dream },
        {"pdf": "undefined_12", "foundry": @system.community },
        {"pdf": "but my", "foundry": @system.ideal },
        {"pdf": "side can get in my way", "foundry": @system.flaw },
        {"pdf": "undefined_5", "foundry": @system.featurebody },
        {"pdf": "undefined_6", "foundry": @system.featureface },
        {"pdf": "undefined_7", "foundry": @system.featurevibe },
        {"pdf": "undefined_8", "foundry": @system.style1 },
        {"pdf": "undefined_9", "foundry": @system.style2 },
        {"pdf": "undefined_10", "foundry": @system.style3 },
        {"pdf": "undefined_11", "foundry": @system.home },

    /* ---------- Abilities ----------*/

        {"pdf": "Abilities01", "foundry": @system.itemTypes.ability[0].name},
        {"pdf": "Abilities02", "foundry": @system.itemTypes.ability[1].name},
        {"pdf": "Abilities04", "foundry": @system.itemTypes.ability[3].name},
        {"pdf": "Abilities05", "foundry": @system.itemTypes.ability[4].name},
        {"pdf": "Abilities06", "foundry": @system.itemTypes.ability[5].name},

    /* ---------- Inventory ----------*/ 
        
        {"pdf": "Inv01", "foundry": @system.itemTypes.item[0].name},
        {"pdf": "Inv02", "foundry": @system.itemTypes.item[1].name},
        {"pdf": "Inv03", "foundry": @system.itemTypes.item[2].name},
        {"pdf": "Inv04", "foundry": @system.itemTypes.item[3].name},

        ]
```

**Comment by bpbarwick** [comment]
I see this is now tagged as not an issue, but as a question. This doesn't seem accurate to me. I have been able to get this module to work with DnD 5e as I have said. I understand how to make it work. It is a bug because it is not working with my supplied PDF template with the Quest RPG system.
